src/ngraph/ops/.clang-format

```diff
@@ -0,0 +1,3 @@
+BasedOnStyle: "Google"
+IndentWidth : 2
+BreakBeforeBraces : Attach
```

Comment:
Are you sure we need this?  AFAIK clang-format will search up the directory tree (relative to the file currently being formatted) until it finds a `.clang-format` file.  See [here](https://stackoverflow.com/a/46374122/387887).

Reply:
I agree with the specification, but it didn't seem to work like that, once I added this, clang-format changed.

Reply:
Odd.  C'est la vie.